3p/integration.js

```diff
@@ -163,6 +163,10 @@ const defaultAllowedTypesInCustomFrame = [
   '_ping_',
 ];
 
+const defaultAllowedTypesWaitForRenderStart = [
```

Comment:
Just `waitForRenderStart`. Add a comment like

List of ad networks that manually call `window.context.renderStart` to emit an event when an ad actually starts rendering. Please add yourself here if you'd like to do so (which we encourage).